cmd/generate/config/rules/config.tmpl

```diff
@@ -63,4 +63,23 @@ regexes = [
     "{{ $stopword }}",{{ end }}
 ]{{ end }}
 {{ end }}
+{{- if and $rule.Verify.URL $rule.Verify.HTTPVerb }}[rules.Verify]
+url = "{{ $rule.Verify.URL }}"
+httpVerb = "{{ $rule.Verify.HTTPVerb }}"
+{{- with $rule.Verify.ExpectedStatus }}
+expectedStatus = [{{ range $j, $status := . }}
+    "{{ $status }}",{{ end }}
+]{{ end }}
+{{- with $rule.Verify.ExpectedBodyContains }}
```

Comment:
It might be overkill but would it make sense to change this to `ExpectedBodyMatches` and have it be a regex? I figure that'll get ahead of some weird more complicated verification situations in the future and make it more flexible if format of responses change a bit.